test/tools/libtesteth/Options.h

```diff
@@ -77,6 +69,7 @@ class Options
 private:
     Options(int argc = 0, const char** argv = 0);
     Options(Options const&) = delete;
+    void setVerbosity(size_t _level);
```

Comment:
It could be static or better just free function in unnamed namespace in Options.cpp

Reply:
Why do you make verbosity parameter `size_t`? Make it `int` like in the rest of the codebase

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es106-dont-try-to-avoid-negative-values-by-using-unsigned